src/cmem/ActivityControl/ActivityControlWidget.tsx

```diff
@@ -102,21 +103,16 @@ export function ActivityControlWidget(props: IActivityControlProps) {
         border,
         canShrink,
         tags,
+        progressFinishedIcon,
     } = props;
-
     const spinnerClassNames = (progressSpinner?.className ?? "") + ` ${eccgui}-spinner--permanent`;
     const widget = (
         <OverviewItem data-test-id={dataTestId} hasSpacing={border} densityHigh={small}>
             {progressBar && <ProgressBar {...progressBar} />}
             {progressSpinner && (
                 <OverviewItemDepiction keepColors>
-                    {progressSpinner?.value ? (
-                        <Icon
-                            name={`state-${progressSpinner.intent}`}
-                            intent={progressSpinner.intent as IntentTypes}
-                            large={!small}
-                            small={small}
-                        />
+                    {progressFinishedIcon && progressSpinner?.value === 1 ? (
+                        progressFinishedIcon
```

Comment:
Pls take care that the icon is displayed in correct size. You probably can use the cloneElement API to force large/small icon property here.